tests/data_files/test-ca_printable.crt

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Could you write how you made these certs in the commit message for "Add tests for relaxed CRL-CA name comparison"?

Reply:
@Patater Well-spotted - we should ideally have instructions in `tests/data_files/Makefile`. In this case, there's no quick way to generate the files in the context of this PR, and instead I benefited from #1641 which allows to use different string encodings (and which extends `tests/data_files/Makefile` significantly along the way). In the sole context of this, PR, though, there's no easy way to generate the new files, which is why I omitted a build description.